features/karaf/openhab-contrib/src/main/feature/feature.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+	Copyright (c) 2010-2019 Contributors to the openHAB project
+
+	See the NOTICE file(s) distributed with this work for additional
+	information.
+
+	This program and the accompanying materials are made available under the
+	terms of the Eclipse Public License 2.0 which is available at
+	http://www.eclipse.org/legal/epl-2.0
+
+	SPDX-License-Identifier: EPL-2.0
+
+-->
+<features name="${project.artifactId}-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+
+    <!-- Features for Java EE bundles -->
+
+    <feature name="openhab-jaxb" description="openHAB JAXB bundles" version="${project.version}">
+        <feature prerequisite="true">wrap</feature>
+        <bundle dependency="true">org.apache.servicemix.specs/org.apache.servicemix.specs.activation-api-1.1/2.9.0</bundle>
+        <bundle dependency="true">mvn:javax.xml.bind/jaxb-api/2.2.12</bundle>
+        <bundle dependency="true">mvn:com.sun.xml.bind/jaxb-impl/2.2.11</bundle>
+        <bundle dependency="true">mvn:com.sun.xml.bind/jaxb-core/2.2.11</bundle>
+    </feature>
+
+    <feature name="openhab-jaxws" description="openHAB JAX-WS (SOAP) bundles" version="${project.version}">
+        <feature prerequisite="true">wrap</feature>
+        <bundle dependency="true">mvn:javax.xml.soap/javax.xml.soap-api/1.4.0</bundle>
+        <bundle dependency="true">wrap:mvn:com.sun.xml.ws/jaxws-rt/2.2.10$Bundle-Name=JAX%20WS%20RI%20Runtime%20Bundle&amp;Bundle-SymbolicName=com.sun.xml.ws.jaxws-rt&amp;Bundle-Version=2.2.10</bundle>
```

Comment:
I checked the feature files provided by Karaf 4.2.4.
The Pax Web feature repo references this bundle: `mvn:org.apache.geronimo.specs/geronimo-jaxws_2.2_spec/1.0`

Just by reading the name of the bundle, it seems to be similar.
Without checking the content myself, does it differ?

Have you checked your other bundle you have choosen that they fit to the ones already used by some of the Karaf stuff (dependent stuff)?

Reply:
will check that